tests/containers/python/lister.py

```diff
@@ -16,7 +18,15 @@ def burner():
         pass
 
 
+def getter():
```

Comment:
If you stay with this function name
```suggestion
def platform_infinite_getter():
```

Reply:
I chose a name that fits the other silly names in the file (`burner` for a function that does nothing and `lister` for a function that calls `os.listdir`)

Reply:
meh ok